share/completions/conda.fish

```diff
@@ -0,0 +1,302 @@
+__ config -l set -x -a "(__fish_conda_config_keys)"        -d "Set a boolean or string key"
+__ config -l remove -x -a "(__fish_conda_config_keys)"     -d "Remove a configuration value from a list key"
+__ config -l remove-key -x -a "(__fish_conda_config_keys)" -d "Remove a configuration key (and all its values)"
+__ config -l stdin                                         -d "Apply configuration given in yaml format from stdin"
+
+# 'help' command
+__ "help"         -d "Displays a list of available conda commands and their help strings"
+__ "help" -x -a "$__fish_conda_commands"
+
+# 'info' command
+__ info -l offline         -d "Offline mode, don't connect to the Internet."
+__ info -s a -l all        -d "Show all information, (environments, license, and system information"
+__ info -s e -l envs       -d "List all known conda environments"
+__ info -s l -l license    -d "Display information about the local conda licenses list"
+__ info -s s -l system     -d "List environment variables"
+__ info -l base            -d "Display base environment path"
+__ info -l unsafe-channels -d "Display list of channels with tokens exposed"
+
+# The remaining commands share many options, so the definitions are written the other way around:
+# the outer loop is on the options
+
+# Option channel
+for cmd in create install remove search update
+    __ $cmd -s c -l channel -d 'Additional channel to search for packages'
+end
+
+# Option channel-priority
+for cmd in create install update
+    __ $cmd -l channel-priority -d 'Channel priority takes precedence over package version'
+end
+
+# Option clobber
+for cmd in create install update
+    __ $cmd -l clobber -d 'Allow clobbering of overlapping file paths (no warnings)'
+end
+
+# Option clone
+__ create -l clone -x -a "(__fish_conda_environments)" -d "Path to (or name of) existing local environment"
+
+# Option copy
+for cmd in create install update
+    __ $cmd -l copy -d 'Install all packages using copies instead of hard- or soft-linking'
+end
+
+# Option download-only
+for cmd in create install update
+    __ $cmd -l download-only -d 'Solve an environment: populate caches but no linking/unlinking into prefix'
+end
+
+# Option dry-run
+for cmd in create install remove update
+    __ $cmd -l dry-run -d 'Only display what would have been done'
+end
+
+# Option file
+for cmd in create install update
+    __ $cmd -l file -d 'Read package versions from the given file'
+end
+
+# Option force
+for cmd in create install remove update
+    __ $cmd -l force -d 'Force install (even when package already installed)'
+end
+
+# Option insecure
+for cmd in create install remove search update
+    __ $cmd -l insecure -d 'Allow conda to perform "insecure" SSL connections and transfers'
+end
+
+# Option mkdir
+for cmd in create install update
+    __ $cmd -l mkdir -d 'Create the environment directory if necessary'
+end
+
+# Option name
+for cmd in create install list remove search update
+    if test $cmd = create
```

Comment:
Move `create` out of the for-loop since there's no shared part.

Reply:
Addressed in a46f05201bdce9d4dfe207e13b5bd64ca715cf0e

Reply:
Addressed by 64d5b94